src/std/net/json-rpc.ss

```diff
@@ -302,7 +302,7 @@
      ;; Use hash so only one of result: or error: shall be printed
      (def result (processor method params))
      (if notification? (void) (hash ("jsonrpc" jsonrpc) ("id" id) ("result" result)))
-     (catch (e) (return-error (if (json-rpc-error? e) e (internal-error)))))))
+     (catch (e) (return-error (if (json-rpc-error? e) e (internal-error e)))))))
```

Comment:
I think we should wrap  `(error-message e)`, the exception could be an arbitrary non serializable object.

Reply:
Done now (I think)